### ISSUE TITLE
Placidity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10213]
**Preferred term label:**
Fear and anger responses that are normally present are blunted

**Synonyms**
Placidity

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C3280828
https://www.ncbi.nlm.nih.gov/medgen/C0564608

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Kluver-Bucy syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@bcoleman96 
How does this fit in?

**Comment by pnrobinson** [comment]
@bcoleman96 
Could this go under 
[Abnormal affect HP:5200261](https://hpo.jax.org/app/browse/term/HP:5200261)

**Comment by pnrobinson** [comment]
Adding for now, we can revise as needed
Adding new term: 
- ID: HP:6000030
- label: Placidity
- parents: [Abnormal affect (HP:5200261)](https://hpo.jax.org/app/browse/term/HP:5200261)
- pmid: PMID:31334941
- def: Flat affect and reduced response to emotional stimuli. Fear and anger responses that are normally present are blunted.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10213
- synonyms:
